script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6.txt

```diff
@@ -27,7 +27,7 @@ It's not about my life at the Arima house. I'm sure it's because I'm just a smal
 "What are you talking about? This is your home now, there is nothing to be afraid of!
 ...#Ah, but you cannot go upstairs, Akiha-sama will be there."
 <Page10>
-"Please feel free to do whatever you look like as long as it is on the first floor.
+"Please feel free to do whatever you want as long as it is on the first floor.
```

Comment:
"do whatever you like" is very Akiha. I actually thought that's what this line said for the first 3 times I read it.

Reply:
I think this is Kohaku speaking, stick or switch?

Reply:
> Please feel free to look around as much as you'd like as long as it is on the first floor.

I like gekimon's